models/repo_permission.go

```diff
@@ -271,6 +271,27 @@ func getUserRepoPermission(e Engine, repo *Repository, user *User) (perm Permiss
 	return
 }
 
+// IsUserRealRepoAdmin check if this user is real repo admin
+func IsUserRealRepoAdmin(repo *Repository, user *User) (bool, error) {
```

Comment:
Can the `Real` in the function name be avoided?

Reply:
``IsUserRepoAdmin`` is exist which is used to check permission(the system administrator will have same permission with repo owner).